### DISCUSSION TITLE
"For more details, you can refer to Test a dataset part in getting_started."

### DISCUSSION THREAD

**Discussion by sakerlrf** [open-mmlab/mmaction2#1167] [General] [answered]
Hello,

In the following link: https://mmaction2.readthedocs.io/en/latest/skeleton_models.html?highlight=test.py , it is stated: 

"For more details, you can refer to Test a dataset part in **[getting_started.](https://mmaction2.readthedocs.io/en/latest/getting_started.md##test-a-dataset)**"

**But, there is no content in the url, yet.** Such an info would be very helpful.

Thank you!

**Comment by dreamerlin** [comment, marked as answer]
https://github.com/open-mmlab/mmaction2/pull/1169
